resources/script/ace-builds/ChangeLog.txt

```diff
@@ -0,0 +1,342 @@
+2015.07.11 Version 1.2.0
```

Comment:
dont put anything in the resources folder besides the pngs. they will not be copied to the phone